features/all/phpcr_query_update.feature

```diff
@@ -153,3 +153,30 @@ Feature: Execute a a raw UPDATE query in JCR_SQL2
             | UPDATE [nt:unstructured] mixin_foo('bar') |
             | UPDATE [nt:unstructured] APPLY mixin_foo('bar') |
             | UPDATE [nt:unstructured] mixin_foo'bar') |
+
+    Scenario Outline: Execute update query with expressions
+        Given I execute the "<query>" command
+        Then the command should not fail
+        And I save the session
```

Comment:
Well, it is a matter of refactoring the context so that ``the command should not fail`` does not take the result of saving the session into account.

And btw, does the user need to save the session explicitly after running a command ? If no, it should not appear in the scenario at all.